gno.land/pkg/gnoland/ugnot/denom.go

```diff
@@ -0,0 +1,11 @@
+package ugnot
+
+import "strconv"
+
+// Denom is the denomination for ugnot, gno.land's native token.
+const Denom = "ugnot"
+
+// ValueString converts `value` to a string, appends "ugnot", and returns it.
+func ValueString(value int64) string {
```

Comment:
Wouldn't it make sense to have a method to do the opposite? Something like :
```go
func ValueInt(value string) (int64, error)
```
Getting the int64 value from a string (or returning an error if the string is invalid). Maybe it exists somewhere else in the codebase, or it might not be needed ?

Reply:
Hi @aeddi, and good question.  `ValueString` is useful because it will append `ugnot` to a number; coin inputs are typically representing as strings -- number + denomination.  Later, this string is parsed using the `ParseCoin` function here: https://github.com/gnolang/gno/blob/6214fe925aa1f955f575f9523ab44f8e274d9c5a/tm2/pkg/std/coin.go#L649